lib/stdlib/src/gen_server.erl

```diff
@@ -916,16 +916,22 @@ init_it(Starter, Parent, Name0, Mod, Args, Options) ->
     CbCache = create_callback_cache(Mod),
     case init_it(Mod, Args) of
 	{ok, {ok, State}} ->
-	    proc_lib:init_ack(Starter, {ok, self()}), 	    
-	    loop(Parent, Name, State, CbCache, infinity, HibernateAfterTimeout, Debug);
-    {ok, {ok, State, TimeoutOrHibernate}}
+	    proc_lib:init_ack(Starter, {ok, self()}),
+	    loop(
+              Parent, Name, State, CbCache, infinity,
+              HibernateAfterTimeout, Debug);
+        {ok, {ok, State, TimeoutOrHibernate}}
           when ?is_timeout(TimeoutOrHibernate);
                TimeoutOrHibernate =:= hibernate ->
-	    proc_lib:init_ack(Starter, {ok, self()}), 	    
-	    loop(Parent, Name, State, CbCache, TimeoutOrHibernate, HibernateAfterTimeout, Debug);
+	    proc_lib:init_ack(Starter, {ok, self()}),
+	    loop(
+              Parent, Name, State, CbCache, TimeoutOrHibernate,
+              HibernateAfterTimeout, Debug);
 	{ok, {ok, State, {continue, _}=Continue}} ->
-	    proc_lib:init_ack(Starter, {ok, self()}), 	    
-	    loop(Parent, Name, State, CbCache, Continue, HibernateAfterTimeout, Debug);
+	    proc_lib:init_ack(Starter, {ok, self()}),
+	    loop(
+              Parent, Name, State, CbCache, Continue,
+              HibernateAfterTimeout, Debug);
```

Comment:
What is the difference? I can only see changes in indentation and line breaks.

Reply:
Yes.  Those are only indentation and line break changes.
There were trailing whitespaces and lines longer than 80, which annoyed me...